.github/workflows/build.yml

```
@@ -0,0 +1,289 @@
+          labels: ${{ steps.activitypub-docker-metadata-private.outputs.labels }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          outputs: type=docker,dest=/tmp/activitypub-${{ matrix.platform_slug }}.tar
+
+      - name: Build Docker Image for Migrations
+        uses: docker/build-push-action@v6
+        with:
+          context: migrate
+          load: true
+          platforms: ${{ matrix.platform }}
+          tags: ${{ steps.migrations-docker-metadata.outputs.tags }}
```

actionlint (1.7.7)

property "migrations-docker-metadata" is not defined in object type {activitypub-docker-metadata-private: {conclusion: string; outcome: string; outputs: {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tags: string; version: string}}; activitypub-docker-metadata-public: {conclusion: string; outcome: string; outputs: {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tags: string; version: string}}; activitypub-migrations-docker-metadata: {conclusion: string; outcome: string; outputs: {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tags: string; version: string}}; activitypub-migrations-docker-metadata-public: {conclusion: string; outcome: string; outputs: {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tags: string; version: string}}}

(expression)